community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DeleteTest.scala

```diff
@@ -34,20 +34,22 @@ class DeleteTest extends DocumentingTestBase with QueryStatisticsTestSupport {
     "Peter"  -> Map("name"->"Peter",  "age" -> 34l)
   )
 
-  override protected def getGraphvizStyle: GraphStyle = 
+  override protected def getGraphvizStyle: GraphStyle =
     AsciiDocSimpleStyle.withAutomaticRelationshipTypeColors()
 
   def section = "Delete"
 
   @Test def delete_single_node() {
+    val createLabeledNode = (db: GraphDatabaseAPI) => db.inTx {
+      db.createNode(DynamicLabel.label("Useless"))
+    } : Unit
```

Comment:
It specifies the return value of the anonymous function. Without it, the return value is inferred to be Node (because that's what db.createNode() returns). I thought it was more clean than to write
`val createLabeledNode: (GraphDatabaseAPI) => Unit = (db: GraphDatabaseAPI) => db.inTx {
      db.createNode(DynamicLabel.label("Useless"))
    }`